tests/FSharp.Compiler.ComponentTests/Conformance/Types/TypeConstraints/IWSAMsAndSRTPs/IWSAMsAndSRTPsTests.fs

```diff
@@ -920,7 +920,10 @@ let main _ =
         |> withNoWarn 3535
         |> withLangVersion70
         |> compile
-        |> shouldSucceed
+        |> shouldFail
+        |> withDiagnostics [
+            (Error 3861, Line 13, Col 71, Line 13, Col 79, "Interfaces cannot access static abstract members directly.")
+        ]
```

Comment:
That's not right, constrained calls `'T.Parse` should work through multiple levels of inheritance, since it will know specific implementation at runtime. `ISpanParsable<'T>.Parse`, if it's non-virtual, shouldn't on other hand.